docs/reference/module-options.md

```diff
@@ -136,4 +138,252 @@ You might use that value as a fallback for user-editable fields.
+- [`autopublish`](#autopublish)
+- [`label`](#label)
+- [`localized`](#localized)
+- [`sort`](#sort)
+- [`slugPrefix`](#slugprefix)
+<!-- - [`contextBar`](#contextbar) -->
+
+### `adminOnly`
+
+<!-- TODO: link to permissions docs when available. -->
+If `true`, only users with the sitewide admin permission can read or write docs of this type. The `@apostrophecms/user` module uses this setting due to its impact on user access. There is no default value.
+
+#### Example
+
+```javascript
+// modules/official-memo/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    adminOnly: true
+  },
+  // ...
+}
+```
+
+### `autopublish`
+
+Set `autopublish` to `true` to automatically publish any changes saved to docs of this type. There is then effectively no draft mode for this doc type.
+
+The core image and file modules use this option, for example. It eliminates the need for users to think about the distinction between draft and published content while preserving the possibility of translation for different locales.
+
+#### Example
+
+```javascript
+// modules/article-category/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    autopublish: true
+  },
+  // ...
+}
+```
+
+<!-- ### `contextBar` -->
+
+<!-- NOTE: Should we keep this on the secrete menu? (not document) -->
+<!-- If `true`, the second row of the admin bar, the "context bar," will be disabled.
+`true` ~ allows the admin bar context bar row to appear -->
+
+### `label`
+
+`label` should be set to a text string to be used in user interface elements related to this doc type. This includes buttons to open piece manager modals and the page type select field.
+
+If not set, Apostrophe will convert the module `name` meta property to a readable label by splitting the `name` on dashes and underscores, then capitalizing the first letter of each word.
+
+#### Example
+
+```javascript
+// modules/feature/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    label: 'Featured Article'
+  },
+  // ...
+}
+```
+
+### `localized`
+
+Defaults to `true`. If set to `false`, this doc type will _not_ be included in the locale system. This means there will be only one version of each doc, regardless of whether multiple locales (e.g., for languages or regions) are active. The "users" piece disables localization in this way.
+
+#### Example
+
+```javascript
+// modules/administrative-category/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    localized: false
+  },
+  // ...
+}
+```
+
+### `sort`
+
+The `sort` option for a doc type defines a sorting order for requests to the database for that type. This setting generally follows the [MongoDB `$sort` aggregation](https://docs.mongodb.com/manual/reference/operator/aggregation/sort/) syntax. The option is set to an object containing field name keys with `1` as a property value for ascending order and `-1` for descending order.
```

Comment:
While its useful to reference the mongoldb doc here, I think it would be more worthwhile just to show these options in our own table, rather than linking to the mongo table and then listing them out.